rm-deployment-status-enhancer-summary/scripts/main.js

```diff
@@ -0,0 +1,78 @@
+
+VSS.ready(function() {
+	var c = VSS.getConfiguration();
+	c.onReleaseChanged(function(release) {
```

Comment:
Why aren't these functions attached via registration of an object via VSS.register ?